src/component/FetchItems.js

```diff
@@ -5,11 +5,17 @@ import { FirestoreCollection } from "react-firestore";
 // need to update timestamp settings with DB
 
 const FetchItems = props => {
+  // Token stored in user's local storage
+  const uniqueToken = localStorage.getItem("uniqueToken");
+
+  // unique DB path based on token
+  const concatPath = "/lists/" + uniqueToken + "/items";
+
   return (
     //fetches specific collection, refreshes every time database chages
     <FirestoreCollection
       //name of collection you want to collect, with filter you can narrow this down to specific document
-      path="items"
+      path={concatPath}
```

Comment:
Non-blocking: Maybe, you could rename `concatPath` to something a bit more descriptive, maybe something like `listItems` or `collectionName` could work!